web/tests/test_quiz_admin.py

```
@@ -0,0 +1,135 @@
+class QuizAdminTests(TestCase):
+    def setUp(self):
+        # Create superuser for admin access
+        self.admin_user = User.objects.create_superuser(
+            username="admin",
+            email="admin@example.com",
+            password="adminpass123"
+        )
+        self.client = Client()
+        self.client.login(username="admin", password="adminpass123")
+        
+        self.captured_queries = CaptureQueriesContext(connection)
```

Ruff (0.8.2)

Undefined name `CaptureQueriesContext`

(F821)